WordleGame/WordleGame.xcodeproj/project.pbxproj

```diff
@@ -20,6 +25,11 @@
 /* End PBXBuildFile section */
 
 /* Begin PBXFileReference section */
```

Comment:
I suggest you create separate folders for keyboard and letter inside your `Models` folders